tests/functional/test_deployment_config.py

```diff
@@ -320,6 +323,47 @@ def test_map_geo_point_prediction(self, extra_model_output_df):
             if extra_model_output_df is not None:
                 assert pred_item["extraModelOutput"] == extra_model_output_df.iloc[index].to_dict()
 
+    def test_map_vector_database_prediction(self, extra_model_output_df):
+        """Verify Vector Database model output"""
+        config = parse_validate_deployment_config_file(self.deployment_config_vector_database)
+        assert config["target"]["name"] == "relevant"
+        assert config["target"]["type"] == "vectordatabase"
+
+        d = {
+            "relevant": [
+                ["relevant text 1"],
+                ["relevant text 2", "relevant text 3"],
+                ["relevant text 1", "relevant text 4", "relevant text 5"],
+            ]
+        }
+        df = pd.DataFrame(data=d)
+        predict_response = PredictResponse(df, extra_model_output_df)
+        response = build_pps_response_json_str(predict_response, config, TargetType.GEO_POINT)
```

Comment:
should this be VDB target-type?
```suggestion
        response = build_pps_response_json_str(predict_response, config, TargetType.VECTOR_DATABASE)
```